src/main/java/cpp/projects/ProjectList.java

```diff
@@ -25,7 +25,7 @@ public ProjectList() {
     public void addProject(String title) {
         System.out.println(Constants.SEPARATOR);
         int index = findProjectIndex(title);
-        if (index == -1) { //this is a new project
+        if (index == Constants.PROJECT_NOT_FOUND) { //this is a new project
```

Comment:
I agree that it is a good idea to take away the magic numbers; this is a good solution